src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { EditButton } from './DashboardTable.styled';
+
+interface DashboardTableProps {
+  isActual: boolean;
+}
+
+function createData(
+  abscenceId: string,
+  type: string,
+  startDate: string,
+  endDate: string,
+  notes: string
+) {
+  return { abscenceId, type, startDate, endDate, notes };
+}
+
+const rows = [
+  createData('1', 'regular', '03/12/2022', '17/12/2022', 'regular'),
+];
+
+const DashboardTable: FC<DashboardTableProps> = ({ isActual }) => {
```

Comment:
this prop is not used. Please don't leave unused props. 
If you plan to use it in the future, create it and use it in the future or now, but don't leave them like that.

Reply:
Fixed